lib/logstash/inputs/jdbc.rb

```diff
@@ -42,6 +42,14 @@
 # `clean_run` is set to true, this value will be ignored and `sql_last_start` will be
 # set to Jan 1, 1970, as if no query has ever been executed.
 #
+# ==== Dealing With Large Result-sets
+#
+# Many JDBC drivers use the `fetch_size` parameter to limit how many results are
+# loaded at a time in the client's cache before retrieving more results from the result 
```

Comment:
how about? > parameter to limit how many results are pre-fetched at a time from the cursor into the client's cache